addons/account/models/chart_template.py

```diff
@@ -776,3 +755,27 @@ def _parse_csv(self, template_code, model, module=None):
             except FileNotFoundError:
                 _logger.debug("No file %s found for template '%s'", model, module)
         return res
+
+    def _load_translations(self, langs=False, companies=False):
+        langs = langs or [code for code, _name in self.env['res.lang'].get_installed()]
+        companies = companies or self.env['res.company'].search([('chart_template', '!=', False)])
+
+        chart2company = {}
+        for company in companies:
+            chart2company.setdefault(company.chart_template, self.env['res.company'])
+            chart2company[company.chart_template] += company
+
+        translation_importer = TranslationImporter(self.env.cr, verbose=False)
+        for chart_template, companies in chart2company.items():
+            template_data = self.env['account.chart.template']._get_chart_template_data(chart_template)
+            del template_data['template_data']
+            for mname, data in template_data.items():
+                for xml_id, record in data.items():
+                    fnames = {fname.split('@')[0] for fname in record}
```

Comment:
Just make sure you understand that:
The `translation_importer.model_translations` expects the `fname` field of `mname` model is a model translated field `translate=True` otherwise, the translations won't be imported.

Note: HTML filed can only be model translated field with both `translate=True`, `sanitize=False` in the code

Reply:
Yes, this is all only for fields with `translate=True`, nothing like HTML (mostly for account and tax names)